src/redux/reducers/data.js

```diff
@@ -1,4 +1,4 @@
-import { SET_LOADED, SET_DATA } from '../actions/сonstants'
+import { SET_DATA, SET_LOADED, SET_WIDTH } from '../actions/constants'
```

Comment:
It might be because of the problem that occured when I pulled changes from master, you might remember that my IDE couldn`t see those methods, that`s why reimported them.

Reply:
I do remember that. I noticed that before